app/assets/javascripts/application.js

```diff
@@ -3,7 +3,7 @@
 //= require jquery_ujs
 //= require jquery-ui
 //= require bootstrap-sprockets
-//= require bootstrap
+//////// require bootstrap
```

Comment:
It doesn't make sense as a comment, so you should just get rid of it if that's the case. @thecristen should know the desired behavior.